docs/index.rst

```diff
@@ -201,6 +185,9 @@ Solving specific problems
 :doc:`topics/jobs`
     Learn how to pause and resume crawls for large spiders.
 
+:doc:`getting-help`
+    Find support beyond the Scrapy documentation.
```

Comment:
What do you think about making this link more prominent somehow? Currently it is in a middle of index, there are 30 topics before it and 10 topics after.

Reply:
Part of my goal putting it there was to try and prevent people from going straight to StackOverflow before reading the relevant documentation. However, given the new page covers that part, I would not mind moving the topic back up.

Maybe at the end of the First steps section?

Reply:
Moved to the end of *First steps*.